libraries/USB/src/USBMIDI.cpp

```diff
@@ -0,0 +1,140 @@
+#include "USBMIDI.h"
+
```

Comment:
Please add check for USB peripheral support as other USB files has.

```suggestion
#if SOC_USB_OTG_SUPPORTED

```